packages/models-library/src/models_library/services_history.py

```diff
@@ -1,19 +1,27 @@
 from datetime import datetime
 from typing import Any, ClassVar, TypeAlias
 
-from models_library.services_base import ServiceKeyVersion
 from pydantic import BaseModel, Field
 
-from .services_types import ServiceVersion
+from .services_types import ServiceKey, ServiceVersion
 from .utils.change_case import snake_to_camel
 
 
+class CompatibleService(BaseModel):
+    key: ServiceKey | None = Field(
+        default=None,
+        description="If None, it refer to current service. Used only for inter-service compatibility",
+    )
+    version: ServiceVersion
+
+
+assert set(CompatibleService.__fields__) == set(CompatibleService.__fields__)  # nosec
+
+
 class Compatibility(BaseModel):
     # NOTE: as an object it is more maintainable than a list
```

Comment:
not sure the note makes sense anymore?

Reply:
haha this is obviously a mistake :-) thx